lib/advisor/core/people/people.ex

```diff
@@ -22,12 +22,14 @@ defmodule Advisor.Core.People do
     end
   end
 
-  def find_group_lead([name: name]) do
-    query = from u in Person, where: u.name == ^name and u.is_group_lead
-    Repo.one(query)
+  def requester(%{requester_id: id}), do: find_by_id(id)
+  def group_lead([name: name]) do
+    Repo.one(from u in Person, where: u.name == ^name and u.is_group_lead)
   end
 
   defp query_by_user(id), do: Repo.one(from u in Person, where: u.id == ^id)
+  defp query_by_name(name), do: Repo.one(from u in Person, where: u.name == ^name)
```

Comment:
`from u in Person, where: u.name == ^name` I think the `u` stand for User, shouldn't be `p` for Person?